src/Display/Tabs/Tabs.test.tsx

```diff
@@ -0,0 +1,116 @@
+import * as React from 'react';
+
+import * as renderer from 'react-test-renderer';
+import '@testing-library/jest-dom/extend-expect';
+import { fireEvent, render } from '@testing-library/react';
+
+import Tabs from './Tabs';
+import { EHorizontalTabVariant } from '../../Utils/StyleConfig';
+import { SecondaryColor } from '../../Style/Colors';
+
+describe('<Tabs/> render', () => {
+  test('should match snapshot', () => {
+    const snapshot = renderer
+      .create(
+        <Tabs>
+          <Tabs.Pane tab="Job">Software Engineer</Tabs.Pane>
+          <Tabs.Pane tab="Company">Glints</Tabs.Pane>
+          <Tabs.Pane tab="Location">Jakarta</Tabs.Pane>
+          <Tabs.Pane tab="Salary">Rp 10,000,000</Tabs.Pane>
+        </Tabs>
+      )
+      .toJSON();
+    expect(snapshot).toMatchSnapshot();
+  });
+});
+
+describe('<Tabs/> should render correctly with the alignment prop', () => {
+  test('horizontal, when alignment is set horizontal', () => {
+    const { getByRole } = render(
+      <Tabs alignment="horizontal">
+        <Tabs.Pane tab="Job">Software Engineer</Tabs.Pane>
+        <Tabs.Pane tab="Company">Glints</Tabs.Pane>
+        <Tabs.Pane tab="Location">Jakarta</Tabs.Pane>
+        <Tabs.Pane tab="Salary">Rp 10,000,000</Tabs.Pane>
+      </Tabs>
+    );
+    const tabContainer = getByRole('tablist');
+    expect(tabContainer).toHaveStyle('display: flex');
+  });
+});
+
+describe('<Tabs/> should render correctly with the variant prop', () => {
+  test('colored, when alignment is set colored', () => {
+    const { getByRole } = render(
+      <Tabs alignment="horizontal" variant={EHorizontalTabVariant.COLORED}>
```

Comment:
same here, test all variants

Reply:
For the tabs, there's no UI for vertical tabs with colored variants, so haven't tested that combination. 
The colored variant exists only for the horizontal one.